modules/homa-lite/src/lib.rs

```diff
@@ -127,7 +127,7 @@ pub mod module {
 
 		/// The fixed cost of withdrawing Staking currency via redeem. In Staking currency.
 		#[pallet::constant]
-		type XcmUnbondFee: Get<Balance>;
+		type HomaUnbondFee: Get<Balance>;
```

Comment:
I think we can move those changes into other smaller PR and have it merged, to reduce the future diff size of this